niworkflows/viz/utils.py

```diff
@@ -105,11 +109,14 @@ def svg_compress(image, compress='auto'):
                     png_b64 = right.split('"')[0]
                     right = '"' + '"'.join(right.split('"')[1:])
 
-                    p = subprocess.run("cwebp -quiet -noalpha -q 80 -o - -- -",
-                                       input=base64.b64decode(png_b64),
-                                       stdout=subprocess.PIPE,
-                                       shell=True, check=True)
-                    webpimg = base64.b64encode(p.stdout).decode('utf-8')
+                    cmd = "cwebp -quiet -noalpha -q 80 -o - -- -"
+                    if PY3:
+                        p = subprocess.run(cmd, input=base64.b64decode(png_b64), shell=True,
+                                           stdout=subprocess.PIPE, check=True).stdout
+                    else:
+                        p = subprocess.check_output(cmd, input=base64.b64decode(png_b64),
+                                                    shell=True, check=True)
```

Comment:
Just so we're clear, we're okay with [requiring 2.7](https://docs.python.org/2/library/subprocess.html#subprocess.check_output)?

And this needs a file-like passed to `stdin` (and `check=True` is redundant if not an error):
```Python
p = subprocess.check_output(cmd, stdin=BytesIO(base64.b64decode(png_b64)),
                            shell=True)
```